01week/rockPaperScissors.js

```diff
@@ -8,11 +8,39 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+/*function rockPaperScissors(hand1, hand2) {
+}*/
 
   // Write code here
+  const rockPaperScissors = (h1, h2) => {
+    if (h1 === h2) {
+     return "Its a tie!"
 
-}
+    } else if (h1 === 'R') {
+      if (h2 === 'P') {
+        return "Paper Wins!" + "<br>" + "You beat me! Nice one!";
+      } else {
+        return "Rock Wins!" + "<br>" + "HAH! I win, you lose.";
+      }
+
+    } else if (h1 ==='P') {
```

Comment:
change all of these to 'rock' , 'paper', or 'scissors', as described in the test.